### PR TITLE
alt click to deflate inflatable walls / doors.

### PR DESCRIPTION
**Pull request by Jey123456**
Alt-click on an inflated one to deflate (same as the deflate option on right click)

### CODE REVIEW

**Comment by Regen1** [comment]
Requesting this to be added to hardsuits deploy helmet as well~

**Comment by Fox-McCloud** [comment]
>Requesting this to be added to hardsuits deploy helmet as well~

That's better suited for an action button, not this.

File: code/game/objects/structures/inflatable.dm [hunk 1]
```diff
@@ -111,6 +111,13 @@
 	if(health <= 0)
 		deflate(1)
 
+/obj/structure/inflatable/AltClick()
+	if(usr.stat || usr.restrained())
+		return
+	if (cheap_hypotenuse(x, y, usr.x, usr.y) >= 2) //allow only 1 dist and diagonals
+		return
```
Comment (TheDZD) [inline code/game/objects/structures/inflatable.dm]: You shouldn't use cheap_hypotenuse for this, use `if(!Adjacent(usr))` instead.

File: code/game/objects/structures/inflatable.dm [hunk 2]
```diff
@@ -111,6 +111,13 @@
 	if(health <= 0)
 		deflate(1)
 
+/obj/structure/inflatable/AltClick()
+	if(usr.stat || usr.restrained())
+		return
+	if (cheap_hypotenuse(x, y, usr.x, usr.y) >= 2) //allow only 1 dist and diagonals
```
Comment (Fox-McCloud) [inline code/game/objects/structures/inflatable.dm]: This isn't the proper way to handle this; you'd want to use the the Adjacent proc.

**Comment by FalseIncarnate** [comment]
Nice quality of life addition. :+1:

Wonder why no one thought of this already...

**Pull request merged**